cmd/liqoctl/cmd/network.go

```diff
@@ -153,18 +149,18 @@ func newNetworkConnectCommand(ctx context.Context, options *network.Options) *co
 		"Type of Gateway Server. Leave empty to use default Liqo implementation of WireGuard")
 	cmd.Flags().StringVar(&options.ServerTemplateName, "server-template-name", forge.DefaultGwServerTemplateName,
 		"Name of the Gateway Server template")
-	cmd.Flags().StringVar(&options.ServerTemplateNamespace, "server-template-namespace", "",
+	cmd.Flags().StringVar(&options.ServerTemplateNamespace, "server-template-namespace", forge.DefaultGwServerTemplateNamespace,
```

Comment:
This was originally left empty as it is defaulted after with provided liqo namespace (to support cases where the user install liqo in a custom namespace).
It is handled here: https://github.com/liqotech/liqo/blob/d0486815e9aa5a06f03f4dc7553089eadd294d8f/pkg/liqoctl/network/handler.go#L322

Same for the ClientTemplateNamespace

Reply:
Understood, so it must be moved in the outer function

Reply:
Correct